fudge-ratio/src/main/webapp/code/time.js

```diff
@@ -1,52 +1,46 @@
 class Timer {
   seconds = 0;
-  minutes = 0;
-  hours = 0;
-
   interval = null;
 
+  constructor(time) {
```

Comment:
A timer constructor so we could to something like:
``const timer = new Timer("00:20:14")``